packages/backend/src/config/app.ts

```diff
@@ -92,3 +99,27 @@ export const Config = {
   /** Frontend **/
   frontendUrl: envString('FRONTEND_URL', 'http://localhost:3000')
 }
+
+function parseRedisTlsConfig(
+  caFile: string,
+  keyFile: string,
+  certFile: string
+): ConnectionOptions | undefined {
+  if (caFile === '' && keyFile === '' && certFile === '') return undefined
+
+  const options: ConnectionOptions = {}
+
+  // self-signed certs.
+  if (caFile !== '') {
+    options.ca = readFileSync(caFile)
+    options.rejectUnauthorized = false
+  }
+
+  // Both are required for client authentication.
+  if (keyFile !== '' || certFile !== '') {
```

Comment:
```suggestion
  if (keyFile !== '' && certFile !== '') {
```
`readFileSync` throws if a path is empty

Or the `readFileSync`s could be moved to their own `if` blocks.